lte/gateway/c/oai/lib/s8_proxy/s8_client_api.cpp

```diff
@@ -455,12 +501,17 @@ static void convert_proto_msg_to_itti_csr(
   } else {
     s5_response->cause = REQUEST_ACCEPTED;
   }
+  if (response.has_protocol_configuration_options()) {
+    get_pco_from_proto_msg(
+        response.protocol_configuration_options(), &s5_response->pco);
+  }
```

Comment:
i can comment the line 499, but shoudnt we be checking `if (response.has_gtp_error())` as the very beginning of this function? 

If we have gtp_error, the other field are going to be bad. Only gtp_error is good. So no need to go through the whole function.